### PR TITLE
Feature/cleanup builders

### PR DESCRIPTION
**Pull request by christianhoff**
**Motivation**
Builder methods have prefixes which blow the code up.

**Changes**
Remove prefixes from builder methods where possible.

### CODE REVIEW

File: src/main/java/org/mqttbee/api/mqtt/mqtt3/message/unsubscribe/Mqtt3UnsubscribeBuilder.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public Mqtt3UnsubscribeBuilder addTopicFilter(@NotNull final MqttTopicFilter top
     }
 
     @NotNull
-    public Mqtt3UnsubscribeBuilder counter(@NotNull final Mqtt3Subscribe subscribe) {
+    public Mqtt3UnsubscribeBuilder copyTopicFiltersFrom(@NotNull final Mqtt3Subscribe subscribe) {
```
Comment (SgtSilvio) [inline src/main/java/org/mqttbee/api/mqtt/mqtt3/message/unsubscribe/Mqtt3UnsubscribeBuilder.java]: I do not like the new name. Maybe is "counterpart" better?

File: src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java [hunk 2]
```diff
@@ -65,26 +68,89 @@ public Mqtt5ClientBuilder(
...
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder executorConfig(@NotNull final MqttClientExecutorConfig executorConfig) {
+        super.executorConfig(executorConfig);
+        return this;
+    }
+
+    @NotNull
+    public Mqtt5ClientBuilder followsRedirects(final boolean followsRedirects) {
```
Comment (SgtSilvio) [inline src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java]: If we stay with "follows" and "allows" then we should also use "usesSsl/Websocket"
or if we stay with "use" then we should also use "follow" and "allow"

File: src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java [hunk 3]
```diff
@@ -65,26 +68,89 @@ public Mqtt5ClientBuilder(
...
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useSsl(@NotNull final MqttClientSslConfig sslConfig) {
+        super.useSsl(sslConfig);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useWebSockets() {
```
Comment (SgtSilvio) [inline src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java]: we should change it to "useWebsocket" as it only uses 1 not many

File: src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java [hunk 4]
```diff
@@ -65,26 +68,89 @@ public Mqtt5ClientBuilder(
...
+        super.useWebSockets();
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useWebSockets(@NotNull final MqttWebsocketConfig websocketConfig) {
```
Comment (SgtSilvio) [inline src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java]: useWebsocket

File: src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java (hunk 2 above)
Comment (christianhoff) [reply src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java]: DONE

File: src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java (hunk 3 above)
Comment (christianhoff) [reply src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java]: DONE

File: src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java (hunk 4 above)
Comment (christianhoff) [reply src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java]: DONE

File: src/main/java/org/mqttbee/api/mqtt/mqtt3/message/unsubscribe/Mqtt3UnsubscribeBuilder.java (hunk 1 above)
Comment (SgtSilvio) [reply src/main/java/org/mqttbee/api/mqtt/mqtt3/message/unsubscribe/Mqtt3UnsubscribeBuilder.java]: I like "reverse" and "revert"

File: src/main/java/org/mqttbee/api/mqtt/mqtt5/message/unsubscribe/Mqtt5UnsubscribeBuilder.java [hunk 5]
```diff
@@ -53,7 +53,7 @@ public Mqtt5UnsubscribeBuilder addTopicFilter(@NotNull final MqttTopicFilter top
     }
 
     @NotNull
-    public Mqtt5UnsubscribeBuilder counter(@NotNull final Mqtt5Subscribe subscribe) {
+    public Mqtt5UnsubscribeBuilder copyTopicFiltersFrom(@NotNull final Mqtt5Subscribe subscribe) {
```
Comment (SgtSilvio) [inline src/main/java/org/mqttbee/api/mqtt/mqtt5/message/unsubscribe/Mqtt5UnsubscribeBuilder.java]: same here

**Pull request merged**